integration-tests/integration/reportPages/enter-incident-details.spec.js

```diff
@@ -19,6 +19,10 @@ context('Submitting details page form', () => {
   const fillFormAndSave = () => {
     const reportUseOfForcePage = ReportUseOfForcePage.visit(offender.bookingId)
     const incidentDetailsPage = reportUseOfForcePage.startNewForm()
+
+    incidentDetailsPage.incidentDate.date().type('12/01/2020{esc}')
```

Comment:
You're joking... Had no idea you could just include esc in the input and it will close it!